pkg/report/report.go

```diff
@@ -0,0 +1,9 @@
+package report
+
+// Report struct
+type Report struct {
```

Comment:
Interessante, mas não preciso de nada com JSON.

Reply:
Apenas para serialização, mas podemos tirar, não afeta em nada.

Reply:
Melhor tirar. É só mais uma dependência pra aumentar a superfície de ataque. Less is more.